client/src/dolbyio_rest_apis/streaming/models/cluster.py

```diff
@@ -2,28 +2,34 @@
 dolbyio_rest_apis.streaming.models.cluster
 ~~~~~~~~~~~~~~~
 
-This module contains the models used by the Cluster model.
+This module contains the models used by the Cluster module.
 """
 
-from dolbyio_rest_apis.core.helpers import get_value_or_default
+from dataclasses import dataclass
+from dataclasses_json import LetterCase, dataclass_json
 
-class Cluster(dict):
-    """The :class:`Cluster` object, which represents a cluster."""
+@dataclass
+class ClusterLocation:
+    """The :class:`ClusterLocation` object, which represents the location of a cluster."""
+
+    city: str | None = None
+    region: str | None = None
+    country: str | None = None
 
-    def __init__(self, dictionary: dict):
-        dict.__init__(self, dictionary)
+@dataclass
+class Cluster:
+    """The :class:`Cluster` object, which represents a cluster."""
 
-        self.id = get_value_or_default(self, 'id', None)
-        self.name = get_value_or_default(self, 'name', None)
-        self.rtmp = get_value_or_default(self, 'rtmp', None)
+    id: str
+    name: str
+    rtmp: str
+    srt: str
+    location: ClusterLocation
```

Comment:
@FabienLavocat missing the features obj
https://docs.dolby.io/streaming-apis/reference/cluster_getclustersinfo

Reply:
✅